README.md

```diff
@@ -22,6 +22,8 @@ Contributors are welcome! Please provide a reproducible test case. Details will
 
 ## Publishing
 
-* `ember github-pages:commit --message "Releasing docs"`
+Release updates to the docsite by running:
+
+* `ember deploy production`
```

Comment:
TravisCI should automatically deploy the docs for tagged versions: https://github.com/adopted-ember-addons/ember-file-upload/blob/54139d922ddbadfdea856dbe799a0abecbf48eac/.travis.yml#L53-L56

Reply:
Have removed the Publishing section.